src/gamefallout3.cpp

```diff
@@ -49,18 +49,18 @@ bool GameFallout3::init(IOrganizer* moInfo)
   return true;
 }
 
-void GameFallout3::checkVariants()
+QString GameFallout3::variantName() const
```

Comment:
You usually want a verb for a function that does something active, like `getCurrentVariant()` or `identifyVariant()`, if there is no verb, the function is usually assumed to have little to no computation or cost, similar to a property or variable. E.g. if `variantName()` just did a map lookup.

In this case there is filesystem access, so not as simple as a variable value.

This is nit-picky stuff, only wrote since you seem to appreciate the feedback